third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -199,49 +200,49 @@ ComputationClient::DataPtr PjRtComputationClient::ReplicateShardedData(
     const ComputationClient::DataPtr& handle) {
   if (PjRtShardedData* sharded_data =
           dynamic_cast<PjRtShardedData*>(handle.get())) {
-      TF_VLOG(1) << "ReplicateShardedData (handle=" << handle->GetOpaqueHandle()
-                 << ", shape=" << handle->shape() << ")";
-      xla::XlaBuilder b("ReplicateShardedData");
-      xla::Shape shape = sharded_data->shape();
-      b.SetSharding(sharded_data->GetSharding());
-
-      // perform a simple identity calculation to reassemble the input as
-      // replicated output.
-      auto x = xla::Parameter(&b, 0, shape, "p0");
-      b.SetSharding(xla::HloSharding::Replicate().ToProto());
-      auto y = xla::Div(x, ConstantR0<float>(&b, 2));
-      auto z = xla::Add(y, y);
-
-      xla::XlaComputation computation =
-          ConsumeValue(b.Build(/*remove_dynamic_dimensions=*/false));
-      xla::ProgramShape program_shape =
-          ConsumeValue(computation.GetProgramShape());
-
-      std::string device = GetDefaultDevice();
-      std::vector<xla::ComputationClient::CompileInstance> instances;
-      instances.push_back({std::move(computation), device,
-                           GetCompilationDevices(device, {}), &shape,
-                           /*should_wrap_parameter=*/false,
-                           /*is_sharded=*/true});
-      std::vector<std::shared_ptr<xla::ComputationClient::Computation>>
-          computations = Compile(std::move(instances));
-
-      auto shards = sharded_data->shards;
-      XLA_CHECK_EQ(shards.size(), GetLocalDevices().size());
-      std::vector<std::vector<ComputationClient::DataPtr>> arguments_by_device(
-          GetLocalDevices().size(), std::vector<ComputationClient::DataPtr>(1));
-      for (auto shard : shards) {
-        std::vector<std::string> device_spec =
-            absl::StrSplit(shard->device(), ':');
-        XLA_CHECK_EQ(device_spec.size(), 2)
-            << "Invalid device specification: " << shard->device();
-        int device_i = std::stoi(device_spec[1]);
-        arguments_by_device[device_i][0] = shard;
-      }
-      xla::ComputationClient::ExecuteReplicatedOptions execute_options;
-      return ExecuteReplicated(*computations.front(), arguments_by_device,
-                               GetLocalDevices(), execute_options)[0][0];
+    TF_VLOG(1) << "ReplicateShardedData (handle=" << handle->GetOpaqueHandle()
```

Comment:
I didn't find anything changed in this part of the code, did I miss anything?

Reply:
`clang-format` reduced the indent here from 4 to 2 spaces to match the rest of the file. Not sure why that didn't get caught in the CI originally